lms/djangoapps/verify_student/views.py

```diff
@@ -579,6 +584,28 @@ def _requirements(self, display_steps, is_active):
 
         return all_requirements
 
+    def _verification_valid_until(self, user, date_format=None):
+        """
+        Check whether the user has a valid or pending verification.
+
+        Arguments:
+            user:
+            date_format: optional parameter, if provided format
+                datetime object in response to string
+
+        Returns:
+            datetime object or datetime object in string format if
+            'date_format' argument is provided
```

Comment:
Do we need this flexibility?  It seems like in our case, we always want the formatted string.  I'd like to avoid returning different types from the same method if possible.